ursula_cli/shell.py

```diff
@@ -192,10 +192,13 @@ def _run_heat(args, hot):
         LOG.error("You must have python-heatclient in your python path")
         raise Exception(e)
 
+    tenant_var = "OS_TENANT_NAME"
+    if 'OS_TENANT_NAME' not in os.environ:
+        tenant_var = 'OS_PROJECT_NAME'
     CREDS = {
         'username': os.environ['OS_USERNAME'],
         'password': os.environ['OS_PASSWORD'],
-        'tenant_name': os.environ['OS_TENANT_NAME'],
+        'tenant_name': os.environ[tenant_var],
```

Comment:
I think you can simplify with just this:

`'tenant_name': os.environ.get('OS_TENANT_NAME', os.environ['OS_PROJECT_NAME'])`

If `OS_TENANT_NAME` key doesn't exist in the environ, the fallback option is used (which also may not exist, so the whole thing should be in a try: except: wrapper but that could be a follow up PR).